### DISCUSSION TITLE
Bun v1.1.1

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#9938] [Announcements]
To install Bun v1.1.1:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.1:
```bash
bun upgrade
```

#### Thank you to 8 contributors!

- [@cirospaciari](https;//github.com/cirospaciari)
- [@dylan-conway](https;//github.com/dylan-conway)
- [@gvilums](https;//github.com/gvilums)
- [@Jarred-Sumner](https;//github.com/Jarred-Sumner)
- [@mangs](https;//github.com/mangs)
- [@nektro](https;//github.com/nektro)
- [@paperdave](https;//github.com/paperdave)
- [@zackradisic](https;//github.com/zackradisic)

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.1.0...bun-v1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.1'>Bun v1.1.1</a>.</em>

**Comment by ZorikovPasha** [comment]
Wow 8 people have fixed 20 bugs and wrote tests only for 4 days, jesus guys, you work like crazy.
